frontend/javascripts/oxalis/model/helpers/nml_helpers.js

```diff
@@ -658,6 +700,44 @@ export function parseNml(
             }
             break;
           }
+          case "userBoundingBox": {
+            const userBoundingBoxId = _parseInt(attr, "id");
```

Comment:
What happens if an NML contains two user bounding boxes with the same id? Should you check this here? Look at the node and tree parsing for inspiration.

Reply:
In fact I choose to ignore the ids later in the code. The `addUserBoundingBoxesAction` update action ensures that the added userBoundingBoxes have a unique id. 
Feel free to look at the code in the `annotation_reducer`.